gp-templates/project-mass-create-sets.php

```diff
@@ -15,12 +15,14 @@
 gp_tmpl_header();
 ?>
 <h2><?php _e( 'Mass-create Translation Sets', 'glotpress' ); ?></h2>
-<p><?php _e( 'Here you can mass-create translation sets in this project.
-The list of translation sets will be mirrored with the sets of a project you choose.
-Usually this is one of the parent projects.', 'glotpress' ); ?></p>
+<p>
+	<?php _e( 'Here you can mass-create translation sets in this project.', 'glotpress' ); ?><br>
+	<?php _e( 'The list of translation sets will be mirrored with the sets of a project you choose.', 'glotpress' ); ?><br>
+	<?php _e( 'Usually this is one of the parent projects.', 'glotpress' ); ?>
```

Comment:
I think these two lines need to stay together without a line break so you know what "this" refers to.

Reply:
So, merging both with a `<br>` tag in the string?

Reply:
No, without any line break.

Reply:
I agree with that. Pushing it in a minute.